net/socket/accept.c

```diff
@@ -284,7 +278,7 @@ int accept(int sockfd, FAR struct sockaddr *addr, FAR socklen_t *addrlen)
   return newfd;
 
 errout_with_socket:
-  sockfd_release(newfd);
+  close(newfd);
```

Comment:
We should not call the close(), since it sets a cancellation point.
```suggestion
  nx_close(newfd);
```

Reply:
Done.